docs/4.3/aws-oss-guide.md

```diff
@@ -173,24 +185,42 @@ An example policy is shown below:
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Sid": "AllAPIActionsOnTeleportAuth",
+            "Sid": "AllAPIActionsOnTeleportClusterState",
             "Effect": "Allow",
             "Action": "dynamodb:*",
-            "Resource": "arn:aws:dynamodb:eu-west-1:123456789012:table/prod.teleport.auth"
+            "Resource": "arn:aws:dynamodb:REGION:ACCOUNTID:table/DYNAMODB-RESOURCE-A"
         },
         {
-            "Sid": "AllAPIActionsOnTeleportStreams",
+            "Sid": "AllAPIActionsOnTeleportClusterStateStreams",
             "Effect": "Allow",
             "Action": "dynamodb:*",
-            "Resource": "arn:aws:dynamodb:eu-west-1:123456789012:table/prod.teleport.auth/stream/*"
-        }
+            "Resource": "arn:aws:dynamodb:REGION:ACCOUNTID:table/DYNAMODB-RESOURCE-A/stream/*"
+        },
+        {
+            "Sid": "AllAPIActionsOnTeleportClusterEvents",
+            "Effect": "Allow",
+            "Action": "dynamodb:*",
+            "Resource": "arn:aws:dynamodb:REGION:ACCOUNTID:table/DYNAMODB-RESOURCE-B"
+        },
+        {
+            "Sid": "AllAPIActionsOnTeleportClusterEventsStreams",
+            "Effect": "Allow",
+            "Action": "dynamodb:*",
+            "Resource": "arn:aws:dynamodb:REGION:ACCOUNTID:table/DYNAMODB-RESOURCE-B/stream/*"
+        },
+        {
+            "Sid": "AllAPIActionsOnTeleportClusterEventIndex",
+            "Effect": "Allow",
+            "Action": "dynamodb:*",
+            "Resource": "arn:aws:dynamodb:REGION:ACCOUNTID:table/DYNAMODB-RESOURCE-B/index/*"
+        },
     ]
 }
 ```
 
 !!! note "Note"
 
-    `eu-west-1:123456789012:table/prod.teleport.auth` will need to be replaced with your DynamoDB instance.
+    `arn:aws:dynamodb:REGION:ACCOUNTID:table/DYNAMODB-RESOURCE-A` and `RESOURCE-B` will need to be replaced with your two DynamoDB instances.
```

Comment:
Should we call these tables instead of instances? I'm not sure what the proper AWS terminology is here, but above we call them tables.

Reply:
💯